src/WindowsUtils/WriteProcessMemoryTest.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/base/casts.h>
+#include <gtest/gtest.h>
+
+#include "OrbitBase/ThreadUtils.h"
+#include "TestUtils/TestUtils.h"
+#include "WindowsUtils/OpenProcess.h"
+#include "WindowsUtils/SafeHandle.h"
+#include "WindowsUtils/WriteProcessMemory.h"
+
+using orbit_windows_utils::WriteProcessMemory;
+
+TEST(WriteProcessMemory, WriteCurrentProcessWithProcessId) {
+  uint32_t process_id = orbit_base::GetCurrentProcessId();
```

Comment:
I would have expected a test to write into a foreign process. Given that there is no `fork` under Windows this might be a bit tricky and not worth it.

Reply:
Yeah, we do the same in other tests, but I'd like to fix that at some point. I added b/226922138.